src/chrome/content/rules/files.tugatech.com.pt.xml

```diff
@@ -1,10 +1,12 @@
 <ruleset name="Files.tugatech.com.pt">
```

Comment:
Please add comments for the mismatch on `blog.tugatech.com.pt`, `magazine.tugatech.com.pt` and `webradio.tugatech.com.pt`